File: Proyecto Python/src/tallerb/Ejercicio7.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package tallerb;

import java.util.Scanner;

/**
 *
 * @author javier
 */
public class Ejercicio7 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        Scanner entrada=new Scanner(System.in);
                
        System.out.print("Ingrese tres números separados por espacios: ");
            int n1=entrada.nextInt(); 
            int n2=entrada.nextInt();
            int n3=entrada.nextInt();
        
        //Se usa condicional para claseficar los números según su orden
         if(n1 < n2 && n2 < n3){
             System.out.print("\n"+n1+", "+n2+", "+n3+". ");
             System.out.println("Los números estan incrementando");
         }
         else if (n1 > n2 && n2 > n3){
             System.out.print("\n"+n1+", "+n2+", "+n3+". ");
             System.out.println("Los números estan disminuyendo");
         }
         else{
             System.out.print("\n"+n1+", "+n2+", "+n3+". ");
             System.out.println("Los números ni se incrementa ni se disminuye");
         }
            
        
        
    }
    
}

File: Proyecto java/Python/Ejercicio7.py
n1 = int(input("Ingrese el primer número: "))
n2 = int(input("Ingrese el segundo número: "))
n3 = int(input("Ingrese el tercer número: "))

#Se usa condicional y se verifica donde se cumple
if n1 < n2 < n3:
    print(f"Los numeros {n1}, {n2}, {n3} estan incrementando")

elif n1 > n2 > n3:
    print(f"Los números {n1}, {n2}, {n3} estan disminuyendo")

else:
    print(f"Los números {n1}, {n2}, {n3} ni se incrementa ni se disminuye")